connectors/sources/confluence.py

```diff
@@ -250,49 +263,66 @@ def get_default_configuration(cls):
                 "order": 3,
                 "type": "str",
             },
+            "data_center_username": {
```

Comment:
Why can't we reuse `username/password` above?

Is it because `depends_on` can't support multiple values?

If so, I'd suggest we add the feature to RCF to support multiple values.

Reply:
Yes, we introduced these new RCFs because `depends_on` does not support multiple value of the same configuration field.